docs/install/docker/README.md

```diff
@@ -0,0 +1,57 @@
+# Docker Install
+
+This version of the FlowForge platform is intended for running in the Docker Container management system. Typically suited for small/medium on premise deployments.
+
+## Prerequistes
+
+### Docker Compose
+
+FlowForge uses Docker Compose to install and manager the required components. Instructions on how to install Docker Compose on your system can be found here:
+
+[https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/)
+
+These instructions assume you are running Docker on a Linux or MacOS host system.
+
+### DNS
+
+The orchestration uses an instance of Nginx to route requests to each Node-RED Project. To do this it needs each instance to have a unique hostname, to generate this the project name is prepended to a supplied domain.
+
+To make this work you will need to configure a DNS server to map a wildcard domain entry to the IP address of the host running Docker. e.g `*.example.com`.
+
+The FlowForge Application will be hosted on `http://forge.example.com`
+
+**Note** When testing locally you can add entries for each project to your `/etc/hosts` file but you must use the external IP address of the host machine, not the loopback address (`127.0.0.1`).
+
+## Installing FlowForge
+
+### Building Contianers
+
+To build the 2 required containers simply run `./build-containers.sh`.
```

Comment:
```suggestion
To build the 2 required containers run `./build-containers.sh`.
```

`simple/simply` are words to avoid (which I don't always succeed in doing myself) as it makes the read feel like it should be obvious when it may not be to them.